node/tests/ui_gateway_test.rs

```diff
@@ -123,9 +130,7 @@ fn daemon_does_not_allow_node_to_keep_his_client_alive_integration() {
     let connected_and_disconnected_assertion =
         |how_many_occurrences_we_look_for: usize, pattern_in_log: fn(port_spec: &str) -> String| {
```

Comment:
Not your fault, just please change `pattern_in_log` to `make_regex_searching_for_port_in_logs`. Thanks

Reply:
renamed